_dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue

```diff
@@ -46,26 +46,68 @@
             class="spinner"
           />
         </div>
-        <b-form-row
-          class="mt-2"
+        <b-container
           v-else
+          class=" maxw-sm-800"
         >
+          <div class="d-flex justify-content-around">
+            <div class="text-center mb-2">
+              <img
+                class="rounded-circle mb-1"
+                src="@/assets/images/google-icon-grey.svg"
+                width="20"
+                height="20"
+              >
+              <p>
+                {{ $t('productFeedSettings.summary.tableHeader1') }}
+              </p>
+            </div>
+            <div :style="cssProps" />
+            <div class="text-center mb-2">
+              <img
+                src="@/assets/images/table-chart.svg"
+                class="mb-1"
+                width="20"
+                height="20"
+              >
+              <p>
+                {{ $t("productFeedSettings.summary.tableHeader2") }}
+              </p>
+            </div>
+          </div>
           <template
             v-for="group in attributesToMap"
           >
-            <b-col
+            <div
               v-for="field in group.fields"
               :key="field.label + group.category"
-              :cols="12"
-              :md="6"
+              class="d-flex col-12 "
```

Comment:
Any reason you don't do ?
```suggestion
              class="row"
```